src/Raven.Server/Documents/Indexes/Static/IndexCompilationCache.cs

```diff
@@ -142,12 +142,6 @@ internal static AbstractStaticIndexBase GenerateIndex(IndexDefinition definition
                     throw new ArgumentOutOfRangeException($"Can't generate index of unknown type {definition.DetectStaticIndexType()}");
             }
 
-            if (configuration.Indexing.StaticIndexingEngineType is SearchEngineType.Corax)
-            {
-                if (index.HasBoostedFields)
-                    throw new IndexCreationException($"{nameof(Corax)} is not supporting boosting inside index yet. Please use Lucene engine.");
```

Comment:
Boosted fields (boosting at fields level) is something that we still don't support (and don't plan to support). Do we have a way to distinguish that case and still throw if index has boosting at fields level?

Reply:
We don't differentiate between field and document boosting during compilation. It will still throw an error from the indexing part. To be more specific: from the converter.